src/HttpMethodsClient.php

```diff
@@ -190,9 +190,9 @@ public function send($method, $uri, array $headers = [], $body = null)
         return $this->sendRequest($this->messageFactory->createRequest(
             $method,
             $uri,
-            '1.1',
             $headers,
-            $body
+            $body,
+            '1.1'
```

Comment:
as this is not an argument to the HttpMethodsClient::send method, i would remove it and let the default happen. otherwise we duplicate default values. i guess it does not matter all that much but would seem cleaner...